src/main/java/org/olf/folio/order/OrderImport.java

```diff
@@ -232,15 +233,19 @@ public  JSONArray  upload(String fileName) throws IOException, InterruptedExcept
 				locations.put(location);
 				 
 				
-				//VENDOR REFERENCE NUMBER IF INCLUDED IN THE MARC RECORD:
+				// as of IRIS release vendorDetail is slightly more complex
 				if (vendorItemId != null) {
```

Comment:
@fereira For some reason this test is true for any file I submit, even if there are no records with a `961$i`. Are you seeing the same?